doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,132 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
 printer_health_offline		| **Optional.** The maximum number of seconds since the last update of cache file before it is considered too old.
 printer_health_multiline	| **Optional.** Multiline output
 
+#### Thola <a id="plugin-contrib-command-thola"></a>
+
+The [Thola](https://thola.io) plugin
+is a tool for monitoring network devices, that mainly uses SNMP.
+To monitor devices the subcommand `thola check` can be used.
+
+`Templates:`
```

Comment:
Also I expect nobody ex. you/us here to utilize these templates, so I see no need to document them separately.

Reply:
Yes, pls.